frontend/src/Components/PostForm.js

```diff
@@ -46,3 +61,9 @@ function PostForm() {
 }
 
 export default PostForm;
+
+const errorStyles = {
```

Comment:
I'm pretty sure we are wanting to style components using BEM for right now, and not using inline styles.

Reply:
Ohh I see.. but this is tiny styling so went I went with Inline